### PR TITLE
Move locale user preferences

### LINKED ISSUE #6660: Move locale selector from slide-in to user preferences
Now that we have the locale selector on the login screen, we should move the locale selector from the slide-in menu to the user preferences page.

### PR DESCRIPTION
**Pull request by bisht-richa**
### Summary
Fixes #6660

### Occurred changes and/or fixed issues
- Moved local from Side nav to preference page 
- Local style should change as per the page location

### How to test

- The login page should have a Local selector
- After logging in side nav should not have a Local selector, it should be on the user preference page
- Style of the Local selector should change as per the page.

### CODE REVIEW

File: shell/pages/prefs.vue [hunk 1]
```diff
@@ -29,6 +30,7 @@ export default {
     menuMaxClusters: mapPref(MENU_MAX_CLUSTERS),
 
     ...mapGetters(['isSingleProduct']),
+    ...mapGetters('i18n', ['selectedLocaleLabel', 'availableLocales']),
```
Comment (richard-cox) [inline shell/pages/prefs.vue:33]: Not needed

File: shell/pages/prefs.vue [hunk 2]
```diff
@@ -153,6 +155,13 @@ export default {
     <div class="mt-10">
       <t k="prefs.theme.autoDetail" :pm="pm" :am="am" />
     </div>
+    <hr />
+    <h4 v-t="'prefs.language'" class="mt-10" />
```
Comment (richard-cox) [inline shell/pages/prefs.vue:159]: `mt-10` isn't needed

Comment (richard-cox) [inline shell/pages/prefs.vue:159]: I'd vote we make this the first entry above `Theme` (if this does move, the `mt-10` should remain)

File: shell/pages/prefs.vue [hunk 3]
```diff
@@ -153,6 +155,13 @@ export default {
...
+    <div class="row">
+      <div class="col span-4">
+        <LocaleSelector></LocaleSelector>
```
Comment (richard-cox) [inline shell/pages/prefs.vue:162]: If there's no content for an element it's usually better to close it within the first tag `<LocaleSelector/>` (same goes for shell/pages/auth/login.vue)

**Review by richard-cox — CHANGES_REQUESTED** [review]

File: shell/pages/prefs.vue (hunk 3 above)
Comment (bisht-richa) [reply shell/pages/prefs.vue:162]: Fixed

File: shell/pages/prefs.vue (hunk 2 above)
Comment (bisht-richa) [reply shell/pages/prefs.vue:159]: fixed

Comment (bisht-richa) [reply shell/pages/prefs.vue:159]: @richard-cox You mean something like this?
Screenshot 2022-09-22 at 14 27 38

**Review by richard-cox — APPROVED** [review]

**Pull request merged**